smart-metastore/src/main/java/org/smartdata/metastore/queries/sort/AuditSortField.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.smartdata.metastore.queries.sort;
+
+public enum AuditSortField implements SortField {
+  ID("id"),
+  USERNAME("username"),
+  TIMESTAMP("timestamp"),
+  OBJECT_TYPE("object_type"),
+  OBJECT_ID("object_id"),
+  OPERATION("operation"),
+  RESULT("result");
+
+  private final String fieldName;
+
+  AuditSortField(String fieldName) {
```

Comment:
No lombok?

Reply:
Added lombok to the similar classes in the `metastore` module